docs/howto/upgrade_from_previous_versions.md

```diff
@@ -0,0 +1,105 @@
+# How to upgrade from previous versions
```

Comment:
> spec.upgrade = 'See docs/howto/UPGRADE.md for upgrade instructions (syntax might not be quite right)

AFAIK there is nothing for docs aside from [adding to RDoc files](http://guides.rubygems.org/specification-reference/#extra_rdoc_files). None of the other doc files are in the ``.gemspec``, should I add this as a start?